addons/sale/static/src/scss/sale_portal.scss

```diff
@@ -29,6 +29,7 @@
 .o_portal .sale_tbody .js_quantity_container {
 
     .js_quantity {
+        width: 3px;
```

Comment:
You should change [it here](https://github.com/odoo/odoo/pull/152270/commits/5010b4aca53d20abca87943a92d1b3473aeb24ba#diff-9915a839a7b8227c3f803926478a3c7e8b4f7ba869ae24999b27dbeb116e49dbL21) like `min-width` --> `width` 
Also 3px it too much tiny :fearful: